ossdbs/api.py

```diff
@@ -209,11 +223,9 @@ def generate_mesh(settings):
         mesh.load_mesh(mesh_settings["LoadPath"])
         return mesh
 
-    if "MeshingHypothesis" in mesh_settings:
-        mesh_hypothesis = mesh_settings["MeshingHypothesis"]
-    else:
-        mesh_hypothesis = {"Type": "Default"}
-    mesh.generate_mesh(mesh_hypothesis)
+    if "MeshingHypothesis" not in mesh_settings:
+        mesh_settings["MeshingHypothesis"] = {"Type": "Default"}
+    mesh.generate_mesh(mesh_settings)
```

Comment:
What is the reasoning behind this change?

Reply:
As the hp-refinement is called in the `mesh` class, we must provide the mesh_settings, not only the mesh_hypothesis.